tools/cli.js

```diff
@@ -133,7 +173,19 @@ commandProcessor.event.help = () => (
   'event <interfaceName> <eventName> [ <arg> [ , ... ] ]'
 );
 
-commandProcessor.connect = (protocol, host, port, appName, callback) => {
+function filterApiCompletions(rawApi) {
+  const api = {};
+  const forbidden = ['_', 'domain'];
+  Object.keys(rawApi).forEach((int) => {
+    api[int] = Object.keys(
+      rawApi[int]).filter(c => !forbidden.some(el => c.startsWith(el)));
```

Comment:
Oh, yes, I did. Thanks for correcting me. It's been about 1.5 years since I switched to Node.js from Python and I still consistently write `all` instead of `every` and `any` instead of `some` in JavaScript 😄 This works the other way around too, though. I tend to be `push`ing elements to Python lists instead of `append`ing them.